x-pack/plugins/index_management/public/store/reducers/row_status.js

```diff
@@ -121,6 +121,7 @@ export const rowStatus = handleActions({
     }, {});
 
     const newState = { ...state };
+    // eslint-disable-next-line guard-for-in
```

Comment:
__question:__ why did ESLint suddenly start to complain about these? Has defaults changed between ESLint 4 and 5?